Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python
+
+COPY . /app
+
+WORKDIR /app
+
+RUN pip install flask
```

Comment:
RUN all of the pip commands in one line, or even better- create a requirements.txt